paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -190,15 +212,12 @@ void CPUQuantizePass::QuantizeConv(Graph* graph,
     GET_IR_NODE_FROM_SUBGRAPH(conv_input, conv_input, conv_pattern);
     GET_IR_NODE_FROM_SUBGRAPH(conv_output, conv_output, conv_pattern);
 
-    // get scales calculated after warmup, they scale variables to MAX=1.0
-    auto scales = Get<VarQuantScale>("quant_var_scales");
-
-    auto input_scale = scales[conv_input->Name()].second.data<double>()[0];
-    bool is_input_unsigned = scales[conv_input->Name()].first;
+    auto input_scale = GetScaleValueForNode(conv_input);
+    bool is_input_unsigned = GetIsScaleUnsignedForNode(conv_input);
```

Comment:
API looks neat but I think you could add a function that returns both to avoid executing twice the same code from GetScaleDataForNode()
Something like:
```
CPUQuantizePass::GetScale***ForNode(const Node* node, double* out_value, bool* out_is_unsigned) const {
  std::pair<.> scale = GetScaleDataForNode(node);
   out_value = scale.second.data<double>()[0];
out_is_unsigned = scale.first;
}
```

Reply:
I have done sth similar.